ggshield/core/config/user_config.py

```diff
@@ -63,14 +64,81 @@ def validate_policy_id(policy_id: str) -> bool:
     return bool(POLICY_ID_PATTERN.fullmatch(policy_id))
 
 
-def validate_policy_ids(values: Iterable[str]) -> None:
-    invalid_excluded_policies = [
-        policy_id for policy_id in values if not validate_policy_id(policy_id)
-    ]
-    if len(invalid_excluded_policies) > 0:
-        raise ValidationError(
-            f"The policies {invalid_excluded_policies} do not match the pattern '{POLICY_ID_PATTERN.pattern}'"
-        )
+@marshmallow_dataclass.dataclass
+class ConfigIgnoredElement(FilteredConfig):
+    comment: Optional[str]
+    until: Optional[datetime]
+
+    @abstractmethod
```

Comment:
`@abstractmethod` on a class which does not inherit from ABC does nothing. But inheriting from ABC here is probably not a good idea since this already inherits from a concrete class.

I would either remove the decorator, or remove the method entirely and re-implement `__str__()` in child classes instead (unless it does not work for some reason).